composer/modules/web/src/plugins/tools/plugin.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year need to be updated

Reply:
updated with commit f29a50ca6573aae3b2ac8e0681ec0d8c47144cfb